### ISSUE TITLE
Recent changes in IRanges

### ISSUE THREAD

**Issue by PeteHaitch** [PeteHaitch/GenomicTuples#13]
- [x] The `findOverlaps` generic now gets an `algorithm` argument 

Herve added this in https://github.com/PeteHaitch/GenomicTuples/commit/c1fc6b6b665f8a40980fe3a3141732be9b8e448c.

However, the building the vignette is broken, at least on my work macbook:

- [x] Fix vignette bug (switched to a HTML vignette in #14)

The PDF vignette builds on work linux server but not on work macbook. Not sure what the problem is. Also, can't build PDF vignettes on Travis - perhaps switch to HTML vignette (see #14).

Update `findOverlaps`-based methods documentation (waiting on corresponding updates to be made in `GenomicRanges`):

- [ ] Specifically, document the `algorithm` argument.

**Comment by PeteHaitch** [comment]
I'm waiting on the `algorithm` argument to be documented in `GenomicRanges`, then I'll just copy this to `GenomicTuples`.

**Comment by PeteHaitch** [comment]
Closing because out-dated and superseded by #24
